samples/bluetooth/bap_broadcast_assistant/src/main.c

```diff
@@ -480,7 +481,8 @@ static void disconnected(struct bt_conn *conn, uint8_t reason)
 
 	printk("Disconnected: %s, reason 0x%02x %s\n", addr, reason, bt_hci_err_to_str(reason));
 
-	bt_conn_unref(broadcast_sink_conn);
+	bt_unpair(BT_ID_DEFAULT, NULL);
+	bt_conn_unref(conn);
```

Comment:
There should only ever be a single connection in this sample, right? Then the change to `unref` seems unnecessary

Reply:
Can we discuss?

Reply:
Sure - For visibility sake it's often better to do it here on Github so others can read the comments later :)